spec/PlayerVsPlayerGameSpec.js

```diff
@@ -10,19 +10,109 @@ describe("PlayerVsPlayerGame", function() {
     game = new PlayerVsPlayerGame(mockUI);
   });
 
-  describe("play", function() {
+  describe("#switchMarker", function() {
```

Comment:
The golden rule about unit tests is that you need to test for every condition. Sometimes this gets cumbersome, which is why we advise smaller tests. This switchMarker function has two possible results, so that should be reflected in the tests.